example.env.service

```diff
@@ -0,0 +1,8 @@
+#####################################
+# Create a .env.service file next to
+# this one with actual values.
+#####################################
+
+SERVER_PORT=9001
+APP_SECURITY_ALLOWACCESS=http://localhost,http://localhost:8080
+APP_PRODUCTS_URL=https://api-dev.library.tamu.edu/project-management/products
```

Comment:
Is this necessary?
I thought APP_PRODUCTS_URL is this system.
Perhaps a copy and paste oversight?

Reply:
Checking now.